src/deadline_worker_agent/installer/install.sh

```diff
@@ -47,13 +47,20 @@ start_service="no"
 telemetry_opt_out="no"
 warning_lines=()
 vfs_install_path="unset"
+python_interpreter_path=unset
 
 usage()
 {
-    echo "Usage: install.sh --farm-id FARM_ID --fleet-id FLEET_ID"
-    echo "                  [--region REGION] [--user USER]"
-    echo "                  [--scripts-path SCRIPTS_PATH]"
+    echo "Usage: install.sh --farm-id FARM_ID"
+    echo "                  --fleet-id FLEET_ID"
+    echo "                  --scripts-path SCRIPTS_PATH"
+    echo "                  --python-interpreter-path PYTHON_INTERPRETER_PATH"
+    echo "                  --region REGION"
```

Comment:
Are these all required arguments now? 

I see logic below that seems to indicate as such. We should mark this change as breaking.

Reply:
>   Are these all required arguments now?

Yes. Both `--scripts-path` and `region` where required before but were not documented here. I just added them now. For `--python-interpreter-path`, yes this is technically a breaking change, but this script is not meant to be invoked directly by end-users. Instead, the `install-deadline-worker` entrypoint is the public interface. It then invokes this bash script and always provides these required arguments.